chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/activity/ActivityMetadata.java

```diff
@@ -34,6 +36,25 @@
     private User responsibleUser;
     private String description;
 
+    @Override
+    public boolean equals(Object another) {
+        if (another == null) {
+            return false;
+        }
+        if (!(another instanceof ActivityMetadata)) {
+            return false;
+        }
+        ActivityMetadata anotherMeta = (ActivityMetadata) another;
+        return ObjectUtils.equals(this.id, anotherMeta.id);
```

Comment:
Is this method deprecated?  Recently, I've found `java.util.Objects` for stuff like this this.
https://commons.apache.org/proper/commons-lang/javadocs/api-3.9/org/apache/commons/lang3/ObjectUtils.html